apps/storefront/app/komponenter/page.tsx

```diff
@@ -0,0 +1,35 @@
+import { ComponentCard } from '@components';
+import React from 'react';
+
+import { data } from './component-list';
+
+const sortedData = data.sort((a, b) => a.title.localeCompare(b.title));
+
+const IS_NEXT_BRANCH = process.env.defaultenvironment === 'next' || 'preview';
```

Comment:
Missing `()` for correct evaluation for `next` or `preview.`
```suggestion
const IS_NEXT_BRANCH = process.env.defaultenvironment === ('next' || 'preview');
```